expressions.js

```diff
@@ -27,9 +27,37 @@ module.exports = {
   },
   // Encore => Vega redirects
   encoreBibPage: {
-      expr: /C__Rb(\d{8})__/,
+    expr: /C__Rb(\d{8})(__|~\$1|$)/,
```

Comment:
This looks good, but if you want to simplify a bit I _think_ you could probably just reduce this to `expr: /C__Rb(\d{8})/`. Yours is more precise for sure, so it's a toss up which is better.

Reply:
i don't mind the extra characters, i think it documents what cases we're expecting.

Reply:
👍🏽